src/main/java/com/lambdatest/tunnel/Tunnel.java

```diff
@@ -538,44 +538,12 @@ public String[] passParametersToTunnelV2(Map<String, String> options) {
 
     public void runCommand(String command) throws IOException {
         try {
-            // ProcessBuilder processBuilder = new ProcessBuilder(command);
-            // System.out.println("Command String: " + command);
             Runtime run = Runtime.getRuntime();
             process = run.exec(command);
             Boolean update = false;
             long start = System.currentTimeMillis();
             long end = start;
-
-            BufferedReader reader = new BufferedReader(new InputStreamReader(process.getInputStream()));
-            String line = null;
-            try {
-                while ((line = reader.readLine()) != null) {
-                    if (line.contains("Downloading update")) {
-                        update = true;
-                    } else {
-                        if (update) {

```

Comment:
removing update workflow as we have deprecated auto update from tunnel binary itself.